src/redux/actions/adminActions.js

```diff
@@ -49,3 +49,5 @@ export const deleteUser = (userId) => (dispatch, getState) => {
         dispatch( returnErrors(err.response.data,err.response.status,"ADMIN_ACTIONS_ERRORS"));
       });
     };
+
```

Comment:
Remove the spaces

Reply:
i am sorry but i dont understand which spaces

Reply:
These spaces